ssl/statem/statem_clnt.c

```diff
@@ -4065,7 +4065,7 @@ int ssl_cipher_list_to_bytes(SSL_CONNECTION *s, STACK_OF(SSL_CIPHER) *sk,
     size_t totlen = 0, len, maxlen, maxverok = 0;
     int empty_reneg_info_scsv = !s->renegotiate
                                 && (SSL_CONNECTION_IS_DTLS(s)
-                                    || s->min_proto_version < TLS1_3_VERSION);
```

Comment:
Here it would be `!s->renegotiate && !SSL_CONNECTION_IS_DTLS(s) && s->min_proto_version <= TLS1_VERSION`